code/game/objects/items/stunbaton.dm

```diff
@@ -257,8 +257,6 @@
 /// After the initial stun period, we check to see if the target needs to have the stun applied.
 /obj/item/melee/baton/proc/apply_stun_effect_end(mob/living/target)
 	var/trait_check = HAS_TRAIT(target, TRAIT_STUNRESISTANCE) //var since we check it in out to_chat as well as determine stun duration
-	if(!target.IsKnockdown())
```

Comment:
This doesn't actually remove the knockdown, it just removes the message. If any baton does apply a knockdown (if ever), they won't be told. You'll need to remove the knockdown callback in the baton effect proc.